### PR TITLE
[cleanup] erefactor/EclipseJdt - Remove unnecessary '$NON-NLS$' tag

### PR DESCRIPTION
**Pull request by cal101**
EclipseJdt cleanup 'RemoveUnnecessaryNlsTag' applied by erefactor.

For EclipseJdt see https://www.eclipse.org/eclipse/news/4.19/jdt.php
For erefactor see https://github.com/cal101/erefactor

### CODE REVIEW

File: org.eclipse.m2e.core.ui/src/org/eclipse/m2e/core/ui/internal/components/PomHierarchyComposite.java [hunk 1]
```diff
@@ -118,7 +118,7 @@ public String getText(Object element) {
       StringBuilder buffer = new StringBuilder();
       Model model = project.getProject().getModel();
       buffer.append(model.getGroupId()).append(" : ") //$NON-NLS-1$
-          .append(model.getArtifactId()).append(" : ") //$NON-NLS-2$
+          .append(model.getArtifactId()).append(" : ")
```
Comment (mickaelistria) [inline org.eclipse.m2e.core.ui/src/org/eclipse/m2e/core/ui/internal/components/PomHierarchyComposite.java:121]: The correction here is incorrect, it should be //$NON-NLS-1$
```suggestion
          .append(model.getArtifactId()).append(" : ") //$NON-NLS-1$
```

File: org.eclipse.m2e.editor/src/org/eclipse/m2e/editor/pom/PomTemplateContext.java [hunk 2]
```diff
@@ -365,7 +365,7 @@ public void addTemplates(MavenProject project, IProject eclipseprj, Collection<T
           //if groupid is the same, suggest ${project.version}
           if(groupId != null && groupId.equals(mvn.getGroupId())) {
             proposals.add(new Template("${project.version}", Messages.PomTemplateContext_project_version_hint, //$NON-NLS-1$
-                contextTypeId, "$${project.version}", false)); //$NON-NLS-3$
+                contextTypeId, "$${project.version}", false));
```
Comment (mickaelistria) [inline org.eclipse.m2e.editor/src/org/eclipse/m2e/editor/pom/PomTemplateContext.java:368]: Also NON-NLS-1

File: org.eclipse.m2e.editor/src/org/eclipse/m2e/editor/pom/PomTemplateContext.java [hunk 3]
```diff
@@ -381,7 +381,7 @@ public void addTemplates(MavenProject project, IProject eclipseprj, Collection<T
               for(String key : keys) {
                 String expr = "${" + key + "}"; //$NON-NLS-1$ //$NON-NLS-2$
                 proposals.add(new Template(expr, Messages.PomTemplateContext_expression_description, contextTypeId,
-                    "$" + expr, false)); //$NON-NLS-2$ //$NON-NLS-1$
+                    "$" + expr, false)); //$NON-NLS-1$
```
Comment (mickaelistria) [inline org.eclipse.m2e.editor/src/org/eclipse/m2e/editor/pom/PomTemplateContext.java:384]: you can also use `'$' + expr` instead and get rid of the NON-NLS here

File: org.eclipse.m2e.editor/src/org/eclipse/m2e/editor/pom/PomTemplateContext.java [hunk 4]
```diff
@@ -400,7 +400,7 @@ public void addTemplates(MavenProject project, IProject eclipseprj, Collection<T
             }
             if(groupId != null && groupId.equals(currentgroupid)) {
               proposals.add(new Template("${project.version}", Messages.PomTemplateContext_project_version_hint, //$NON-NLS-1$
-                  contextTypeId, "$${project.version}", false)); //$NON-NLS-3$
+                  contextTypeId, "$${project.version}", false));
```
Comment (mickaelistria) [inline org.eclipse.m2e.editor/src/org/eclipse/m2e/editor/pom/PomTemplateContext.java:403]: NON-NLS-1

Comment (cal101) [reply org.eclipse.m2e.editor/src/org/eclipse/m2e/editor/pom/PomTemplateContext.java]: All fixed.

**Comment by mickaelistria** [comment]
Thanks!

**Comment by cal101** [comment]
@mickaelistria You took the burden of reviewing so I took the burden of opening the project in eclipse. ;-)

**Comment by mickaelistria** [comment]
> You took the burden of reviewing so I took the burden of opening the project in eclipse. ;-)

Sounds like a good and fair deal ;)

**Comment by mickaelistria** [comment]
Note that @HannesWell should be thanked for really making opening the projects in Eclipse IDE less and less of a burden these days. I'm pretty certain some of the recent changed he made saved a few minutes and a couple of annoying troubleshooting.

**Comment by cal101** [comment]
I noticed the changes and it sounds well thought out but I couldn't follow.
So If any manual step to be done changed I didn't noticed.
I am ignoring all errors in eclipse, make the changes and compile on command line.

**Pull request merged**